aiidalab_widgets_base/codes.py

```diff
@@ -33,43 +33,38 @@ class CodeDropdown(ipw.VBox):
     """
     selected_code = Union([Unicode(), Instance(Code)], allow_none=True)
     codes = Dict(allow_none=True)
+    allow_hidden_codes = Bool(False)
+    allow_disabled_computers = Bool(False)
 
-    def __init__(self,
-                 input_plugin,
-                 text='Select code:',
-                 hidden_codes=False,
-                 disabled_computers=False,
-                 path_to_root='../',
-                 **kwargs):
+    def __init__(self, input_plugin, text='Select code:', path_to_root='../', **kwargs):
```

Comment:
The `text` parameter is usually described as `description` for most ipywidgets.

Reply:
Ok